apps/bookmark/style/save.css

```diff
@@ -69,10 +69,21 @@ body[data-mode="put"] #edit-bookmark-header {
   text-transform: uppercase;
 }
 
+#app-icon, #app-icon-placeholder {
+  display: block;
+  width: 6rem;
+  height: 6rem;
+  margin: 0.5rem auto 0 auto;
+}
+
+#app-icon.hidden, #app-icon-placeholder.hidden {
```

Comment:
What do you think about creating a rule like

```css
.hidden {
  display: none;
}
```

removing https://github.com/benfrancis/gaia/blob/1134795/apps/bookmark/style/save.css#L50 and these two new rules?